bench_test.go

```diff
@@ -79,6 +79,8 @@ func BenchmarkSendMessage(b *testing.B) {
 			wgReg.Add(1)
 			wgCh.Add(1)
 
+			name := name
```

Comment:
What are you trying to fix here? This was done in a purpose, the exact one that you pointed is the doc, it's not wrong.